kolibri/core/content/tasks.py

```diff
@@ -353,6 +355,26 @@ def automatic_resource_import():
     process_content_requests()
 
 
+@register_task(
+    long_running=True,
+    status_fn=get_status,
+)
+def automatic_synchronize_content_requests_and_import():
+    """
+    Task that synchronizes content requests for all facilities/datasets on the device and enqueues the automatic_resource_import task afterwards.
+    - Calls synchronize_content_requests for all facilities/datasets on the device.
+    - Enqueues the automatic_resource_import task after synchronizing content requests.
+    """
+
```

Comment:
One safeguard could be to check the device setting here and finish the task early if it has been changed since it enqueued?